tidb-lightning/deploy-tidb-lightning.md

```diff
@@ -73,69 +73,7 @@ aliases: ['/docs-cn/dev/tidb-lightning/deploy-tidb-lightning/','/docs-cn/dev/ref
 
 ## 部署 TiDB Lightning
 
-本节介绍 TiDB Lightning 的两种部署方式：[使用 TiDB Ansible 部署](#使用-tidb-ansible-部署-tidb-lightning)和[手动部署](#手动部署-tidb-lightning)。
```

Comment:
All removal of TiDB Ansible from TiDB Lightning docs are confirmed with @kennytm .

Reply:
@kennytm Can we deploy TiDB Lightning using TiUP now?